pages/index.tsx

```diff
@@ -1,44 +1,46 @@
 /* Copyright 2020 the Deno authors. All rights reserved. MIT license. */
 
-import React, { useEffect, useState } from "react";
+import React from "react";
 import Head from "next/head";
 import Link from "next/link";
 import CodeBlock from "../components/CodeBlock";
 import Footer from "../components/Footer";
-import { entries } from "../util/registry_utils";
 import versions from "../versions.json";
 import { NextPage, GetStaticProps } from "next";
 import InlineCode from "../components/InlineCode";
 import Header from "../components/Header";
 
-interface SimpleEntry {
-  name: string;
-  desc: string;
-}
+// interface SimpleEntry {
+//   name: string;
+//   desc: string;
+// }
+
 interface HomeProps {
-  thirdPartyEntryPool: SimpleEntry[];
+  // thirdPartyEntryPool: SimpleEntry[];
   latestStd: string;
 }
 
-const NUM_THIRD_PARTY = 12;
-const POOL_NUM_THIRD_PARTY = 50;
+// const NUM_THIRD_PARTY = 12;
+// const POOL_NUM_THIRD_PARTY = 50;
 
-const Home: NextPage<HomeProps> = ({ thirdPartyEntryPool, latestStd }) => {
+const Home: NextPage<HomeProps> = ({
+  /* thirdPartyEntryPool ,*/ latestStd,
+}) => {
   const complexExampleProgram = `import { serve } from "https://deno.land/std@${latestStd}/http/server.ts";
 const s = serve({ port: 8000 });
 console.log("http://localhost:8000/");
 for await (const req of s) {
   req.respond({ body: "Hello World\\n" });
 }`;
 
-  const [thirdPartySelection, setThirdPartySelection] = useState<
-    SimpleEntry[] | null
-  >(null);
-  useEffect(() => {
-    setThirdPartySelection(
-      RandomEntriesFromArray(thirdPartyEntryPool, NUM_THIRD_PARTY)
-    );
-  }, []);
+  // const [thirdPartySelection, setThirdPartySelection] = useState<
+  //   SimpleEntry[] | null
+  // >(null);
+  // useEffect(() => {
+  //   setThirdPartySelection(
+  //     RandomEntriesFromArray(thirdPartyEntryPool, NUM_THIRD_PARTY)
+  //   );
+  // }, []);
```

Comment:
Consider removing dead code

Reply:
I wanna re-add this in the future, but I can remove it for now.